app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -217,9 +218,15 @@ class Core(private val context: Context) {
 
     /**
      * Shared Preferences that encrypt/decrypt using Android KeyStore and lib-dataprotect for 23+
-     * otherwise simply stored
+     * only on Nightly/Debug for now, otherwise simply stored.
+     * See https://github.com/mozilla-mobile/fenix/issues/8324
      */
-    fun getSecureAbove22Preferences() = SecureAbove22Preferences(context, KEY_STORAGE_NAME)
+    fun getSecureAbove22Preferences() =
+        SecureAbove22Preferences(
+            context = context,
+            name = KEY_STORAGE_NAME,
+            forceInsecure = !Config.channel.isNightlyOrDebug
+        )
```

Comment:
What will happen to release users that have been using this before? 🤔

Reply:
All the features using secure storage were only enabled in Nightly (just Logins).
I have also added a check that attempts to migrate data from the other store (but won't crash if we can't access the keystore)
https://github.com/mozilla-mobile/android-components/pull/5902